internal/db/db.go

```diff
@@ -50,6 +51,30 @@ func (db *dB) InsertCompose(jobId, accountId, orgId string, request json.RawMess
 	return err
 }
 
+// Returns (true, nil) if the composeId belongs to the accountId (false, nil)
+// otherwise.
+// Returns (false, err) when an error occurs
+func (db *dB) DoesComposesBelongsToAccount(accountId string, composeId string) (
```

Comment:
Alright then, I'm gonna make a special vim configuration. I tend to prefer to have 80 cols because with this I can have more vertical splits, and the more I have, the better I feel ^^